### PR TITLE
[DOR-140] Add "Error: " prefix to the page title when error

### PR DESCRIPTION
**Pull request by arturopala**

### CODE REVIEW

File: it/uk/gov/hmrc/traderservices/controllers/CreateCaseJourneyISpec.scala [hunk 1]
```diff
@@ -599,6 +657,32 @@ class CreateCaseJourneyISpec extends CreateCaseJourneyISpecSetup with TraderServ
...
+        )
+        journey.setState(state)
+        givenAuthorisedForEnrolment(Enrolment("HMRC-XYZ", "EORINumber", "foo"))
+
+        val payload = Map("freightType" -> "")
+
+        val result = await(request("/pre-clearance/export-questions/transport-type").post(payload))
+
+        result.status shouldBe 200
+        result.body should include(htmlEscapedPageTitle("view.export-questions.freightType.title"))
```
Comment (Tasmine10ten) [inline it/uk/gov/hmrc/traderservices/controllers/CreateCaseJourneyISpec.scala:681]: htmlEscapedPageTitleWithError instead?

File: it/uk/gov/hmrc/traderservices/controllers/CreateCaseJourneyISpec.scala [hunk 2]
```diff
@@ -715,6 +791,35 @@ class CreateCaseJourneyISpec extends CreateCaseJourneyISpecSetup with TraderServ
...
+        givenAuthorisedForEnrolment(Enrolment("HMRC-XYZ", "EORINumber", "foo"))
+
+        val payload = Map(
+          "vesselName" -> "Foo Bar"
+        )
+
+        val result = await(request("/pre-clearance/export-questions/vessel-info-required").post(payload))
+
+        result.status shouldBe 200
+        result.body should include(htmlEscapedPageTitle("view.export-questions.vessel-details.title"))
```
Comment (Tasmine10ten) [inline it/uk/gov/hmrc/traderservices/controllers/CreateCaseJourneyISpec.scala:818]: As above - htmlEscapedPageTitleWithError

File: app/uk/gov/hmrc/traderservices/views/WaitingForFileVerificationView.scala.html [hunk 3]
```diff
@@ -27,7 +27,7 @@
 @(successAction: Call, failureAction: Call, checkStatusAction: Call, backLink: Call)(implicit request: Request[_], messages: Messages)
 
 @govukLayout(
-  pageTitle = Some(messages("view.upload-file.waiting")),
+  pageTitle("view.upload-file.waiting"),
```
Comment (Tasmine10ten) [inline app/uk/gov/hmrc/traderservices/views/WaitingForFileVerificationView.scala.html:30]: It seems some of the other upload pages might need this change - error prefix doesn't prompt on /file-upload or /file-uploaded (both new & amend)

File: it/uk/gov/hmrc/traderservices/controllers/CreateCaseJourneyISpec.scala (hunk 2 above)
Comment (Tasmine10ten) [reply it/uk/gov/hmrc/traderservices/controllers/CreateCaseJourneyISpec.scala:818]: Looks like the error tests from here on need this change

File: app/uk/gov/hmrc/traderservices/views/WaitingForFileVerificationView.scala.html (hunk 3 above)
Comment (arturopala) [reply app/uk/gov/hmrc/traderservices/views/WaitingForFileVerificationView.scala.html:30]: there are no errors on /file-uploaded?

**Review by Tasmine10ten — APPROVED** [review]

**Pull request merged**